src/routes.ts

```diff
@@ -50,6 +51,9 @@ const routesToSanitize: string[] = [
 
   // Storage
   '/storage/:bucket/:path*',
+  
+  // Firealerts
+  `/firealerts/`,
```

Comment:
Does this really need to be sanitized? I don't see identifiers (or even parameters) in the URL

Reply:
Fixed